api_tests/files/views/test_file_list.py

```diff
@@ -1,138 +1,160 @@
-# -*- coding: utf-8 -*-
-from nose import tools as nt
-
 from framework.auth.core import Auth
```

Comment:
This should go with local imports & the pytest import should be at the top of the file